assets/scripts/segments/__tests__/Segment.test.js

```diff
@@ -203,7 +204,7 @@ describe('Segment', () => {
         }
       )
       setLastStreet() // ToDo: needs to be refactored
-      fireEvent.mouseOver(screen.getByTestId('segment'))
+      userEvent.hover(screen.getByTestId('segment'))
       fireEvent.keyDown(document, { key: 'Delete' })
```

Comment:
Let's leave a comment on this `fireEvent` so we know why it could not be changed to `userEvent` this time. Also please reference this issue in the comment for later reference: https://github.com/testing-library/user-event/issues/506